### DISCUSSION TITLE
How to change output type parameter in NextJS build command?

### DISCUSSION THREAD

**Discussion by LucasStbnr** [vercel/next.js#56774] [Help] [answered]
### Summary

I want to change the output type parameter in the NextJS build command so that it only outputs 'standalone' during Docker builds and not locally. 
Is there a command similar to ```next build --output standalone``` that can accomplish this?

**Comment by icyJoseph** [comment, marked as answer]
Not really, but maybe you can use an `env` var, and set the output mode in `next.config.js` to `standalone`, if the env var has a given value, otherwise, keep it `undefined`

**Reply by LucasStbnr** [reply]
Thank you it worked for me
